tests/transport/spdp/spdp_transport.cpp

```diff
@@ -0,0 +1,452 @@
+// Test for the mini-transport in SPDP (sequence number handling)
+
+#include "dds/DCPS/transport/rtps_udp/RtpsUdpInst.h"
+#ifdef ACE_AS_STATIC_LIBS
+#include "dds/DCPS/transport/rtps_udp/RtpsUdp.h"
+#endif
+
+#include "dds/DCPS/transport/framework/NetworkAddress.h"
+
+#include "dds/DCPS/RTPS/BaseMessageTypes.h"
+#include "dds/DCPS/RTPS/GuidGenerator.h"
```

Comment:
`GuidGenerator.h` apparently doesn't have header guards and it looks like pragma once doesn't look like it works on safety profile, so it conflicts from being included in `RtpsDiscovery.h`. I would suggest adding header guards to `GuidGenerator.h`.